roles/rundeck/tasks/rundeck-cli.yaml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Download the Rundeck CLI inside the container
+  ansible.builtin.shell: podman exec '{{ rundeck_container_name }}' wget {{ rundeck_cli_url }}
+
+- name: Unpack the Rundeck CLI inside the container
+  ansible.builtin.shell: podman exec '{{ rundeck_container_name }}' sudo unzip -qqo {{ rundeck_cli_url | basename }} -d {{ rundeck_cli_container_path }}
```

Comment:
sudo?????

Reply:
Changed approach so no horrible sudo inside the container nónense anymore